doc/classes/Timer.xml

```diff
@@ -37,14 +38,17 @@
 			<argument index="0" name="paused" type="bool">
 			</argument>
 			<description>
-				Set whether the timer is paused or not. A paused timer will be inactive until it is unpaused again.
+				Pause the timer. If [code]paused[/code] is [code]true[/code], the timer will not process until it is started or unpaused again, even if [code]start[/code] is called.
```

Comment:
Pause -> Pauses

Use method links: change "[code]start[/code]" to [method start]

Reply:
Ah, this was not found within http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html - maybe this documentation should be updated too.

Reply:
http://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html#improve-formatting-with-bbcode-style-tags